pythonforandroid/archs.py

```diff
@@ -62,6 +62,8 @@ def get_env(self, with_flags_in_cc=True):
 
         if self.ctx.ndk == 'crystax':
             env['LDFLAGS'] += ' -L{}/sources/crystax/libs/{} -lcrystax'.format(self.ctx.ndk_dir, self.arch)
+            env['CFLAGS'] += ' -I{}/sources/python/{}/include/python/'.format(
+                self.ctx.ndk_dir, self.ctx.python_recipe.version[0:3])
```

Comment:
I believe other recipes should benefit from it.

Reply:
This is now probably addressed in https://github.com/kivy/python-for-android/pull/1361